### DISCUSSION TITLE
Community Spotlight: Madza

### DISCUSSION THREAD

**Discussion by BekahHW** [deepgram/community#18] [General help]
We love our community members :tada: and Madza has been an amazing one! [Madza](https://github.com/madzadev) is a Technical writer for LogRocket and SitePoint and a frontend developer who loves to build side projects. He’s passionate about audio as a hobby, and you can can see how he's blended his interests in tech and audio with his Deepgram Hackathon Project.

Madza was one of our runner-ups for the [Deepgram + Dev Hackathon](https://dev.to/devteam/join-us-for-a-new-kind-of-hackathon-on-dev-brought-to-you-by-deepgram-2bjd). His project, VoiceCue is an app that uses Deepgram Speech-to-Text technology to create cue timecodes to identify things like sentiments, entities, and tags. With his app, the user can use the transcript to identify different features and visualize where they are in the recording. There’s even a Custom Search function!

You can check out all the details here:

- Link to Code: [GitHub - madzadev/voice-cue: 📣 Find sentiments, tags, entities, and actions in your voice recordings instantly](http://github.com/madzadev/voice-cue)  🧑‍💻
- Deployed app: cue.madza.dev 🎉
- Link to blog post: [Introducing VoiceCue - Find sentiments, tags, entities, actions like a DJ 🧙✨](https://dev.to/madza/introducing-voicecue-find-sentiments-tags-entities-actions-like-a-dj-4kjk) 

 And you can listen to our [Twitter Space with Madza](https://twitter.com/i/spaces/1yNxaNablXjKj?s=20) Friday, September 9, at 1:30pm ET.
 
 Thanks for being our first Community Spotlight, Madza! If you have any questions for him or have positive words for Madza, please share them in the comments!

**Comment by michaeljolley** [comment]
That was a really cool project @madzadev! We should hook you up with our beta API endpoints so you can try to use some upcoming features for sentiment, tags and entities. That could really simplify the project!

**Reply by madzadev** [reply]
Thank you so much for the kind words @MichaelJolley  ! 👍✨💯

**Comment by phazonoverload** [comment]
I remember being really impressed when I first saw your project @madzadev!

**Reply by madzadev** [reply]
Means the world, thanks for being so supportive! 👍✨💯
